core/src/test/java/com/sequenceiq/cloudbreak/controller/validation/network/MultiAzValidatorTest.java

```diff
@@ -0,0 +1,97 @@
+package com.sequenceiq.cloudbreak.controller.validation.network;
+
+import static org.mockito.ArgumentMatchers.anyString;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.cloudbreak.common.network.NetworkConstants;
+import com.sequenceiq.cloudbreak.domain.stack.instance.InstanceGroup;
+import com.sequenceiq.cloudbreak.domain.stack.instance.network.InstanceGroupNetwork;
+import com.sequenceiq.cloudbreak.validation.ValidationResult;
+
+@ExtendWith(MockitoExtension.class)
+public class MultiAzValidatorTest {
+
+    @InjectMocks
+    private MultiAzValidator underTest;
+
+    @Mock
+    private ValidationResult.ValidationResultBuilder builder;
+
+    @BeforeEach
+    void setUp() {
+        ReflectionTestUtils.setField(underTest, "suporterMultiAzVariants", Set.of("AWS_NATIVE"));
+        underTest.initSupportedVariants();
+    }
+
+    @Test
+    public void testAwsNativeWhenOneSubnetUsedShouldNotCauseAnyValidationError() {
+        String variant = "AWS_NATIVE";
+        Set<InstanceGroup> instanceGroups = Set.of(
+                instanceGroup(Set.of("subnet-123")),
+                instanceGroup(Set.of("subnet-123"))
+        );
+        underTest.validateMultiAzForStack(variant, instanceGroups, builder);
+        Mockito.verify(builder, Mockito.times(0)).error(anyString());
+    }
+
+    @Test
+    public void testAwsNativeWhenMultipleSubnetUsedShouldNotCauseAnyValidationError() {
+        String variant = "AWS_NATIVE";
+        Set<InstanceGroup> instanceGroups = Set.of(
+                instanceGroup(Set.of("subnet-123", "subnet-145")),
+                instanceGroup(Set.of("subnet-123"))
+        );
+        underTest.validateMultiAzForStack(variant, instanceGroups, builder);
+        Mockito.verify(builder, Mockito.times(0)).error(anyString());
+    }
+
+    @Test
+    public void testAwsWhenOneSubnetUsedShouldNotCauseAnyValidationError() {
+        String variant = "AWS";
+        Set<InstanceGroup> instanceGroups = Set.of(
+                instanceGroup(Set.of("subnet-123", "subnet-145")),
+                instanceGroup(Set.of("subnet-123"))
+        );
+        underTest.validateMultiAzForStack(variant, instanceGroups, builder);
+        Mockito.verify(builder, Mockito.times(1)).error(anyString());
+    }
```

Comment:
The body seems to be a copy of `testAwsWhenMultipleSubnetUsedShouldCauseAnyValidationError`, so the name does not reflect the expectation.

Reply:
Ping, `testAwsWhenOneSubnetUsedShouldNotCauseAnyValidationError()` and `testAwsWhenMultipleSubnetUsedShouldCauseValidationError()` still have the exact same logic.